packages/members-api/lib/controllers/router.js

```diff
@@ -233,8 +241,16 @@ module.exports = class RouterController {
         }
 
         if (member.related('products').length !== 0) {
-            res.writeHead(403);
-            return res.end('No permission');
+            const subscriptions = await member.related('stripeSubscriptions').fetch();
```

Comment:
Can we just replace this whole check with `if (member.status !== 'paid')` ?

Reply:
🤔 Hmm yea we could actually, the whole thing is essentially to not allow checkout session for paid members.